app/components/fastboot-only/body-bottom.js

```diff
@@ -48,6 +50,13 @@ export default Component.extend({
 		}, simpleStore));
 	}),
 
+	asyncScriptsPath: computed(function () {
```

Comment:
what will happen if someone goes from glee.wikia.com/de to glee.wikia.com/pl?
I think we should reload a page right?

Reply:
I added a test case to make sure it happens in https://github.com/Wikia/mobile-wiki/pull/618/commits/23fecfede869717bebada9f3020202837806c2b2